frontend/components/common/CompatibleDevice.tsx

```diff
@@ -114,32 +110,29 @@ export function CompatibleDevice({
                </Text>
             )}
             {!truncateModels && hiddenVariants.length > 0 && (
-               <details ref={detailsRef}>
-                  <Summary
-                     mt="2px"
-                     lineHeight="short"
-                     fontSize="xs"
-                     fontWeight="medium"
-                     color="brand.500"
-                     cursor="pointer"
+               <details>
+                  <summary
```

Comment:
Could we keep the style props by using `<chakra.summary>` here?

Reply:
Yes we could. How is that different than using the summary element as you described here though? https://github.com/iFixit/react-commerce/pull/1604#discussion_r1171940067 Besides the fact that we aren't defining it in a separate file.